docs/hugo/.htmltest.yml

```diff
@@ -1,3 +1,13 @@
 DirectoryPath: public
 CheckExternal: false
```

Comment:
We do want to check the external links as well?

Reply:
I'll do that in a separate PR.